### PR TITLE
ARC-1519-part-1 - Improve Jira Routing for connect/app creation 

### PR DESCRIPTION
**Pull request by krazziekay**
**What's in this PR?**
- Updating Jira Configuration to Jira
- Added Test cases

**How has this been tested?**  
- In local dev environment
- Test cases

### CODE REVIEW

File: src/routes/github/configuration/github-configuration-get.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Installation } from "models/installation";
 import { Subscription } from "models/subscription";
 import { NextFunction, Request, Response } from "express";
-import { getInstallations, InstallationResults } from "../../jira/configuration/jira-configuration-get";
+import { getInstallations, InstallationResults } from "../../jira/jira-get";
```
Comment (joshkay10) [inline src/routes/github/configuration/github-configuration-get.ts:4]: should be able to update this route to `"routes/jira/jira-get"`

Comment (krazziekay) [reply src/routes/github/configuration/github-configuration-get.ts]: updated

**Review by joshkay10 — APPROVED** [review]

**Comment by joshkay10** [comment]
Looking good, just make sure that they are given a blitz in stage or if you want once all 50 PRs are approved put them all in stage together and a bunch of us can try all the various routes + some soaking in stage.

**Comment by krazziekay** [comment]
> Looking good, just make sure that they are given a blitz in stage or if you want once all 50 PRs are approved put them all in stage together and a bunch of us can try all the various routes + some soaking in stage.

I've created a separated branch `ARC-1519-final`, which would be the base branch. I'll merge all these parts into that base branch and can test the branch at once later on in staging.

**Pull request merged**